CMakeLists.txt

```diff
@@ -81,8 +105,7 @@ endif()
 set(IRODS_BUILD_WITH_WERROR ${IRODS_BUILD_WITH_WERROR_DEFAULT} CACHE BOOL "Choose whether to compile with -Werror.")
 
 find_package(nlohmann_json "3.6.1" REQUIRED)
-find_package(fmt "8.1.1"
-  HINTS "${IRODS_EXTERNALS_FULLPATH_FMT}")
+find_package(fmt "8.1.1")
```

Comment:
Should this be marked as `REQUIRED`?

Reply:
Is that line effectively a no-op then?

Reply:
Yes

Reply:
Is there any reason we can't remove it now?

Reply:
After thinking about it, we should keep this line, and add `REQUIRED`, as the icommands themselves use fmt. This is the best practice when consuming a target that might also be provided as a dependency by a different target. This avoids problems should the other target make changes.

Reply:
Sounds good to me.

Reopening so you can mark when it is resolved.